### PR TITLE
[16076] Bump version to v2.8.1

### PR DESCRIPTION
**Pull request by EduPonz**
* Add v2.8.1 release notes
* Bump version

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/49/)

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/50/)

**Comment by EduPonz** [comment]
@richiprosima please test this. CI is failling due to connectivity issues. Test pass locally

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/51/)

File: docs/notes/notes.rst [hunk 1]
```diff
@@ -1,15 +1,16 @@
 Version 2.8.0
```
Comment (JLBuenoLopez-eProsima) [inline docs/notes/notes.rst:1]: Update here the version

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/52/)

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM. CI should be fixed. A similar issue was detected in another product recently

**Pull request merged**